model/Check/System/LocalNamespaceCheck.php

```diff
@@ -40,10 +40,14 @@ protected function doCheck(): Report
         $uriProvider = $this->getUriProviderService();
         $namespace = trim($uriProvider->getOption('namespace'), ' #');
```

Comment:
Stupid one: though it's not 100% valid case - TAO will continue working (most of its parts, haven't checked it fully, but backoffice works fine), when `LOCAL_NAMESPACE` also has `#` at the end. So, we need to be careful with trimming `#` out of namespace configured in `uriProvider.conf.php`.
p.s.
Though I'm not even 80% sure that smth should be changed in here. It's more a "warning" than a real problem

Reply:
It's the good point. I think we can add 
`if ($namespace !== trim(LOCAL_NAMESPACE, ' #')) {` 
Just to be sure that those values (without #) will be equal.
WDYT?

Reply:
Interesting one. IMO - we can use it like that. :+1: